src/app/pages/resources/overview/overview.jsx

```diff
@@ -411,36 +400,41 @@ function ChplResourcesOverview() {
               </TableRow>
             </TableHead>
             <TableBody>
-              { getOrgs(acbQuery, 'acbs').map((acb) => (
+              {getOrgs(acbQuery, 'acbs').map((acb) => (
                 <TableRow key={acb.id}>
                   <TableCell>ONC-ACB</TableCell>
                   <TableCell>{acb.name}</TableCell>
                   <TableCell>{acb.acbCode}</TableCell>
                   <TableCell>
                     {acb.website
-                     && <ChplLink href={acb.website} />}
+                      && <ChplLink href={acb.website} />}
                   </TableCell>
                 </TableRow>
               ))}
-              { getOrgs(atlQuery, 'atls').map((atl) => (
+              {getOrgs(atlQuery, 'atls').map((atl) => (
                 <TableRow key={atl.id}>
                   <TableCell>ONC-ATL</TableCell>
                   <TableCell>{atl.name}</TableCell>
                   <TableCell>{atl.atlCode}</TableCell>
                   <TableCell>
                     {atl.website
-                     && <ChplLink href={atl.website} />}
+                      && <ChplLink href={atl.website} />}
                   </TableCell>
                 </TableRow>
               ))}
             </TableBody>
           </Table>
-          <InternalScrollButton
-            name="Back to the top"
-            id="main-content"
-            analytics={{ event: 'Jump to top of Overview', category: 'Navigation' }}
-          />
-          <Divider />
+          <div >
+            <Fab
```

Comment:
If this doesn't work, are you asking me to fix it? I can't merge and deploy non-working stuff

Reply:
So i wasnt sure how to apply an anchor link to this the FAB? Do we have an id, we could point the link to so the page scrolls back up? Looks like you took this out until we get it fixed?